source/acquaman/IDEAS/IDEASXASScanActionController.cpp

```diff
@@ -177,12 +177,11 @@ void IDEASXASScanActionController::buildScanControllerImplementation()
 			scan_->addAnalyzedDataSource(newRegion, false, true);
 			detector->addRegionOfInterest(region);
 
-			QString regionName = regionAB->name().replace(" ","_");
-			AM1DCalibrationAB* normalizedRegion = new AM1DCalibrationAB(QString("Norm%1").arg(regionName));
-			normalizedRegion->setDescription(QString("Norm%1").arg(regionName));
+			AM1DCalibrationAB* normalizedRegion = new AM1DCalibrationAB(QString("norm_%1").arg(newRegion->name()));
```

Comment:
newRegion->name() is the regionAB->name() without space (`regionAB->name().remove(' ')`). 
the removed `regionName` is the regionAB->name() with `_` replacing space (`regionAB->name().replace(" ", "_")`). 

Therefore, the changes is slightly different with the original ones (name without `_`). Is this what we want?